scripts/ingests/ingest_utils.py

```diff
@@ -1214,3 +1234,76 @@ def ingest_instrument(db, telescope=None, instrument=None, mode=None):
                 raise SimpleError(msg + '\n' + str(e))
 
     return
+
+
+def get_gaiadr3(gaia_id, verbose=True):
+    # Currently setup just to query one source
+    # TODO: add some debug and info messages
+    gaia_query_string = f"SELECT " \
+                        f"parallax, parallax_error, " \
+                        f"pmra, pmra_error, pmdec, pmdec_error " \
+                        f"phot_g_mean_flux, phot_g_mean_flux_error, phot_g_mean_mag," \
+                        f"phot_rp_mean_flux, phot_rp_mean_flux_error, phot_rp_mean_mag" \
+                        f"FROM gaiadr3.gaia_source WHERE " \
+                        f"gaiadr3.gaia_source.source_id = '{gaia_id}'"
+    job_gaia_query = Gaia.launch_job(gaia_query_string, verbose=verbose)
+
+    gaia_data = job_gaia_query.get_results()
```

Comment:
Add a handle here for if something breaks in the launch job or gets results, either raise a SimpleError or pass and leave a debug message, depending on intended use case